### PR TITLE
Bugfix: OcTableFiles selection model

### PR DESCRIPTION
**Pull request by paulcod3**
## Description
We used a local prop to hold the state of the selection model of octablefiles which turned out to be wrong.
Should be fixed with this PR.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [x] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation added/updated

### CODE REVIEW

File: src/components/table/OcTableFiles.vue [hunk 1]
```diff
@@ -630,16 +629,6 @@ export default {
             type: "folder",
             size: "5324435",
             mdate: "Mon, 11 Jan 2021 14:34:04 GMT"
-          },
```
Comment (pascalwengerter) [inline src/components/table/OcTableFiles.vue:633]: Why delete this example?

**Review by pascalwengerter — CHANGES_REQUESTED** [review]
1. Missing changelog item (minor thing)
2. Question as commented
3. Works great in general, but was able to cause a weird issue:

- Select multiple files via checkbox
- Open Details via Contextmenu
- Unselect all items via the "Clear selection/X" button
- OcTableFiles only shows `name` and `quickactions` but not `size` and `last modified` anymore 🤷🏽

File: src/components/table/OcTableFiles.vue [hunk 2]
```diff
@@ -384,11 +383,12 @@ export default {
       this.$emit(EVENT_FILE_DROPPED, fileId)
     },
     addSelectedResource(file) {
-      const selectedResourceInResources = this.selectedResources.some(e => e.id === file.id)
+      const selectedResourceInResources = this.selection.some(e => e.id === file.id)
```
Comment (kulmann) [inline src/components/table/OcTableFiles.vue:386]: Seeing this again now, what do you think about `isSelected` als name for the const?

File: src/components/table/OcTableFiles.vue (hunk 1 above)
Comment (kulmann) [reply src/components/table/OcTableFiles.vue:633]: If this was deleted in the gist I sent to you @paulcod3 it was not intentional. Please restore. ;-)

**Review by kulmann — CHANGES_REQUESTED** [review]
As discussed, add the PR URL to the already existing `enhancement` changelog item. That's sufficient, as this code with the highlighting-changes was never released. IMO dedicated bugfix changelogs make sense after something was released before.

Bug that Pascal mentions is weird and needs to be double checked. I will also take the time and run this in your open web PR soonish (on Monday at the latest).

**Comment by paulcod3** [comment]
Ok found the solution to the following two issues:
1. Sidebar wasn't going to open when selecting a file
2. When opening the sidebar and deselecting all table columns went missing

As a solution we need to refactor some web logic since the "highlightedFile" getter isn't valid anymore.
@kulmann I would discuss the solution when you have time since it affects existing logic

**Review by kulmann — APPROVED** [review]
LGTM 👍

**Pull request merged**